src/components/Photo/SelfieIntro.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import { h, Component } from 'preact'
+import PageTitle from '../PageTitle'
+import style from './style.css'
+import theme from '../Theme/style.css'
+import classNames from 'classnames'
+import Button from '../Button'
+import { localised } from '../../locales'
+import { trackComponent } from '../../Tracker'
+import withCrossDeviceWhenNoCamera from '../Capture/withCrossDeviceWhenNoCamera'
+import withCameraDetection from '../Capture/withCameraDetection'
+import { compose } from '~utils/func'
+
+const InstructionPure = ({instruction}) => (
```

Comment:
I don't think this shoud be a separate component. Also, it's more understandable when key is set during the iterration then on the component (not entirely sure that it even does the trick?)